domains/shan.json

```diff
@@ -8,5 +8,6 @@
     },
     "record": {
         "CNAME": "adwaith-shan-pk.github.io"
+        "TXT": "dh=edc2f8a1ed533b91a794ced9244cb4fa8a138531"
```

Comment:
You can't have other records with `CNAME`, and for discord verification https://www.is-a.dev/docs/discord_verification/